src/NuGet.Core/NuGet.Versioning/docs/README.md

```diff
@@ -0,0 +1 @@
+NuGet's implementation of Semantic Versioning.
```

Comment:
````suggestion
# NuGet.Versioning

Install the `NuGet.Versioning` to use NuGet's implementation of [Semantic Versioning](https://semver.org/).

## Examples

```cs
NuGetVersion version = NuGetVersion.Parse("1.2.3-prerelease");

Console.WriteLine($"Major version: {version.Major}"); // 1
Console.WriteLine($"Minor version: {version.Minor}"); // 2
Console.WriteLine($"Patch version: {version.Patch}"); // 3

// Get the normalized version string that is unique to the identity of this version:
Console.WriteLine($"Normalized version: {version.ToNormalizedString()}");
```
````